### PR TITLE
SDK requirements.txt file & new config modules

### PR DESCRIPTION
**Pull request by pranavrd**

### CODE REVIEW

File: configs/config-example.json [hunk 1]
```diff
@@ -1,74 +1,110 @@
...
-            "auditingDatabaseName": "defaultdb",
-            "auditingDatabaseUserName": "immudb",
-            "auditingDatabasePassword": "password",
-            "auditingDatabaseTableName": "tableName",
-            "auditingPoolSize": 25
-            }
-
-    ]
-}
+{
```
Comment (abhi4578) [inline configs/config-example.json:1]: Hey,  thie contains lot of test config options, which is not needed in actual deployment. You can maybe split it into another called ``config-test.json`` which as test config options and one ``config-example.json`` which does not.

File: configs/config-example.json [hunk 2]
```diff
@@ -1,74 +1,110 @@
...
+  "version": "1.0",
+  "zookeepers": [
+    "zookeeper"
+  ],
+  "clusterId": "iudx-catalogue-cluster",
+  "host": "server",
```
Comment (abhi4578) [inline configs/config-example.json:7]: This option is no longer required . Its shifted to auth verticle

File: configs/config-example.json [hunk 3]
```diff
@@ -1,74 +1,110 @@
...
+      "ratingIndex": "",
+      "databaseUser": "",
+      "databasePassword": "",
+      "databasePort": 1234,
+      "bypassAuth": true
+    },
+    {
+      "id": "iudx.catalogue.server.authenticator.AuthenticationVerticle",
+      "verticleInstances": 2,
+      "keystorePath": "",
```
Comment (abhi4578) [inline configs/config-example.json:27]: keystore is no longer need in auth verticle

File: configs/config-example.json [hunk 4]
```diff
@@ -1,74 +1,110 @@
...
+      "keystorePassword": "",
+      "authServerHost": "",
+      "dummyToken": "",
```
Comment (abhi4578) [inline configs/config-example.json:30]: This is test config option

File: configs/config-example.json [hunk 5]
```diff
@@ -1,74 +1,110 @@
...
+      "host": "",
+      "token": "",
```
Comment (abhi4578) [inline configs/config-example.json:32]: This is test config option

File: configs/config-example.json [hunk 6]
```diff
@@ -1,74 +1,110 @@
...
+      "cert": "",
```
Comment (abhi4578) [inline configs/config-example.json:33]: is this ``cert`` or ``publicKey`` ? previous config had latter.

File: configs/config-example.json [hunk 7]
```diff
@@ -1,74 +1,110 @@
...
+      "databasePort": 1234,
+      "databasePassword": "",
+      "databaseUser": "",
+      "docIndex": ""
+    },
+    {
+      "id": "iudx.catalogue.server.apiserver.ApiServerVerticle",
+      "keystorePath": "",
+      "keystorePassword": "",
+      "token": "authtest.iudx.io/md.adil@datakaveri.org/f16fbf39871014085fbb74de43b39bea",
```
Comment (abhi4578) [inline configs/config-example.json:49]: This is test config

File: configs/config-example.json [hunk 8]
```diff
@@ -1,74 +1,110 @@
...
+      "admin_token": "authtest.iudx.io/md.adil@datakaveri.org/f16fbf39871014085fbb74de43b39bea",
```
Comment (abhi4578) [inline configs/config-example.json:50]: test config option

**Review by abhi4578 — CHANGES_REQUESTED** [review]

File: configs/config-example.json (hunk 6 above)
Comment (pranavrd) [reply configs/config-example.json:33]: it's cert

**Comment by abhi4578** [comment]
ok to test

**Pull request merged**